src/commands/dora/api.ts

```diff
@@ -0,0 +1,45 @@
+import type {AxiosPromise, AxiosRequestConfig, AxiosResponse} from 'axios'
+
+import {getRequestBuilder} from '../../helpers/utils'
+
+import {DeploymentEvent} from './interfaces'
+
+export const datadogSite = process.env.DATADOG_SITE || process.env.DD_SITE || 'datadoghq.com'
```

Comment:
since it's a new command, let's get rid of deprecated `DATADOG_SITE`
```suggestion
export const datadogSite = process.env.DD_SITE || 'datadoghq.com'
```